.gitignore

```diff
@@ -7,3 +7,4 @@ packages/docs/.next
 packages/docs/out
 packages/docs/pages/Dev
 .idea
+.DS_Store
```

Comment:
This is specific to macos (same for the `.idea` being specific to an editor), they should probably be setup as global `ignores`.

https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer